xml/deploy_cephadm.xml

```diff
@@ -797,15 +796,17 @@ o- / ............................................................... [...]
   o- cephadm_bootstrap ............................................. [...]
   | o- advanced .................................................... [...]
   | o- ceph_conf ................................................... [...]
+  | o- ceph_image_path ............... [registry.suse.com/ses/7/ceph/ceph]
   | o- dashboard ................................................... [...]
   |   o- force_password_update ................................. [enabled]
   |   o- password ................................... [randomly generated]
   |   o- username ................................................ [admin]
   | o- mon_ip ..................................................... [None]
   o- containers .................................................... [...]
-  | o- images ...................................................... [...]
-  |   o- ceph ........................ [registry.suse.com/ses/7/ceph/ceph]
-  | o- registries ................................................ [empty]
+  | o- auth ........................................................ [...]
```

Comment:
Should be:
![Screenshot from 2020-09-14 09-40-52](https://user-images.githubusercontent.com/14297426/93063629-7354e380-f66e-11ea-8e85-d8867055a3c1.png)